### ISSUE TITLE
ApplleNVMe Assert Failed: 0 == (status)

### ISSUE THREAD

**Issue by NickWoodhams** [RehabMan/patch-nvme#6]
Generated the HackrNVM kext for Sierra using your latest update. Injecting with Clover. What's strange is that my NVMe drives load but this message is continually spammed on my logs. 

Unloading and reloading the kext on an AHCI drive didn't get rid of the error.
Installing in S/L/E didn't work either. 

Any ideas?

I have 2 950Pros, 1 SM951 (AHCI), and 1 XP941 (AHCI)

default 22:43:25.969568 -0400   kernel  AppleNVMe Assert failed: 0 == (status)
default 22:43:25.969576 -0400   kernel  Exit
default 22:43:25.969578 -0400   kernel  file: /BuildRoot/Library/Caches/com.apple.xbs/Sources/IONVMeFamily/IONVMeFamily-234.1.16/AppleNVMeRequest.cpp
default 22:43:25.969584 -0400   kernel  line: 690

![screen shot 2016-09-26 at 10 54 47 pm](https://cloud.githubusercontent.com/assets/1296347/18858771/83accad4-843c-11e6-83ca-76edd06cdbee.png)

**Comment by RehabMan** [comment]
As per README, you must remove IONVMeFamily.kext from /S/L/E.
Always read carefully.

**Comment by NickWoodhams** [comment]
@RehabMan I appreciate your hard work. I did read the entire README carefully. I previously removed that kext per your README. I also rebuilt the cache and tried loading without caches as well. Loading HackrNVM with Clover injection, does the same thing when the patched kext is installed to S/L/E. To clarify, my NVMe drives DO WORK but it is is just the error in my syslog that is the issue. 
![screenshot 2016-09-27 01 46 05](https://cloud.githubusercontent.com/assets/1296347/18861558/2d3d7a28-8454-11e6-9c98-c6a3aa838b1a.png)
